lms/templates/lti_launches/new_lti_launch.html.jinja2

```diff
@@ -1 +1,17 @@
-<iframe width="100%" height="100%" src="{{hypothesis_url}}" />
\ No newline at end of file
+<script>
+  function receiveMessage(event) {
+    if (event.origin !== "http://localhost:5000") {
+      return;
+    }
+
+    if (event.data !== "gimme_grant_token") {
+      return;
+    }
+
+    // Send back grant token in response.
+    var grantToken = '{{ grant_token }}';
+    event.source.postMessage(grantToken, event.origin)
```

Comment:
As noted in hypothesis/client#764 (comment) we should probably implement a JSON-RPC message protocol instead of using `grantToken` directly as the message.